frontend/src/metabase/query_builder/components/AggregationPopover/AggregationPopover.jsx

```diff
@@ -52,10 +52,6 @@ export default class AggregationPopover extends Component {
     // passing a dimension disables the field picker and only shows relevant aggregations
     dimension: PropTypes.object,
 
-    // DEPRECATED: replaced with `query`
-    tableMetadata: PropTypes.object,
-    datasetQuery: PropTypes.object,
-
```

Comment:
Not used directly. Also there's no props spread, so child components don't seem to need them either